sagemaker_studio_image_build/data/buildspec.template.yml

```diff
@@ -9,6 +9,9 @@ phases:
       - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION --registry-ids 217643126080)
       - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION --registry-ids 727897471807)
       - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION --registry-ids 626614931356)
+      - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION --registry-ids 683313688378)
+      - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION --registry-ids 520713654638)
+      - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION --registry-ids 462105765813)
```

Comment:
Please add a comment on what Deep Learning Container or Algorithm container these correspond to.

Reply:
Please see the details below:
- 683313688378 - [Prebuilt Amazon SageMaker Docker Images for Scikit-learn and Spark ML in us-east-1](https://docs.aws.amazon.com/sagemaker/latest/dg/pre-built-docker-containers-scikit-learn-spark.html)
- 520713654638 & 462105765813 - [Amazon SageMaker RL Containers](https://github.com/aws/sagemaker-rl-container)